importer/api/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""
+Import OLX data into LORE.
+"""
+
+# stdlib
+from glob import glob
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join
+import tarfile
+import zipfile
+
+# PyPi
```

Comment:
I think it is safe to leave these comments out, but it is worth noting that we should follow the three import group convention, of core, third-party and internal app